ui/src/message_popup/author/edit_question_page.jsx

```diff
@@ -11,15 +11,96 @@ export default React.createClass({
   displayName: 'EditQuestionPage',
 
   propTypes: {
-    questionId: React.PropTypes.string.isRequired
+    questionId: React.PropTypes.string.isRequired,
+    loaded: React.PropTypes.bool,
+    allQuestions: React.PropTypes.object
+  },
+
+  getDefaultProps(){
```

Comment:
I used this format of `getDefaultProps` for the files that load the questions from the database so that we can write tests for them with props instead of just having everything in the state.